setup.py

```diff
@@ -72,7 +72,9 @@
             'feature_extraction.audio_transfer.DistilAudioTransfer = distil.primitives.audio_transfer:AudioTransferPrimitive',
             'data_preprocessing.audio_loader.DistilAudioDatasetLoader = distil.primitives.audio_loader:AudioDatasetLoaderPrimitive',
             'clustering.k_means.DistilKMeans = distil.primitives.k_means:KMeansPrimitive',
-            'data_transformation.data_cleaning.OutputDataframe = distil.primitives.output_dataframe:OutputDataframePrimitive'
+            'data_transformation.data_cleaning.OutputDataframe = distil.primitives.output_dataframe:OutputDataframePrimitive',
+            'data_transformation.list_encoder.DistilListEncoder = distil.primitives.list_encoder:ListEncoderPrimitive',
```

Comment:
These names are defined in the d3m core under `metadata.primitive_names`.  The d3m core package will check to see if your name exists in that list, and will log a warning if it can't find it.  It will still load and run though, so its not critical.  Its too late for us to PR the d3m core this time around, but we should try to update the next time they open it up for changes.